### PR TITLE
Use the quay.io repo everywhere instead of rancher

### PR DESCRIPTION
**Pull request by skitt**
Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Comment by skitt** [comment]
This ends up pushing `:dev` images, I need to fix that.

File: scripts/release [hunk 1]
```diff
@@ -4,12 +4,13 @@ set -e
...
-docker tag rancher/dapper-base:dev quay.io/submariner/dapper-base:${DOCKER_TAG#"v"}
-docker tag rancher/dapper-base:dev quay.io/submariner/dapper-base:"${TRAVIS_COMMIT:0:7}"
-for i in $(docker images --format "{{.Repository}}:{{.Tag}}" | grep "quay.io/submariner"); do docker push $i; done
+docker tag ${REPO}/submariner:dev ${REPO}/submariner:${DOCKER_TAG#"v"}
+docker tag ${REPO}/submariner:dev ${REPO}/submariner:"${TRAVIS_COMMIT:0:7}"
+docker tag ${REPO}/submariner-route-agent:dev ${REPO}/submariner-route-agent:${DOCKER_TAG#"v"}
+docker tag ${REPO}/submariner-route-agent:dev ${REPO}/submariner-route-agent:"${TRAVIS_COMMIT:0:7}"
+docker tag ${REPO}/dapper-base:dev ${REPO}/dapper-base:${DOCKER_TAG#"v"}
+docker tag ${REPO}/dapper-base:dev ${REPO}/dapper-base:"${TRAVIS_COMMIT:0:7}"
+for i in $(docker images --format "{{.Repository}}:{{.Tag}}" | grep -E "${REPO}/.*:(${DOCKER_TAG#v}|${TRAVIS_COMMIT:0:7})"); do docker push $i; done
```
Comment (dimaunx) [inline scripts/release]: Does the grep -E is really required? Seems like grep "quay.io/submariner" and grep -E "${REPO}/.*:(${DOCKER_TAG#v}|${TRAVIS_COMMIT:0:7})" return the same results that include all tags not just latest and commit hash. Maybe grep -E "${REPO}/.*:(${DOCKER_TAG#v}|\${TRAVIS_COMMIT:0:7})" will return just the required tags.

Comment (dimaunx) [reply scripts/release]: :) github remove the \ character  before $,  i meant grep -E "${REPO}/.:(tag|\tag)"

Comment (skitt) [reply scripts/release]: On my system:

```
$ docker images --format "{{.Repository}}:{{.Tag}}" | grep "quay.io/submariner"
quay.io/submariner/submariner-route-agent:latest
quay.io/submariner/submariner:latest
quay.io/submariner/submariner-operator:dev
quay.io/submariner/submariner-operator:0.0.1
```

whereas

```
$ docker images --format "{{.Repository}}:{{.Tag}}" | grep -E "quay.io/submariner/.*:(latest|blah)"
quay.io/submariner/submariner-route-agent:latest
quay.io/submariner/submariner:latest
```

Comment (dimaunx) [reply scripts/release]: Yes you are correct i didn't set TRAVIS_COMMIT and it returned all tags.

**Pull request merged**